src/ui/default-prop-types.js

```diff
@@ -286,7 +286,7 @@ export const mediaLibraryItemShape = PropTypes.shape({
 
 export const mediaLibraryItemWithRelevanceShape = PropTypes.shape({
   ...mediaLibraryItemProps,
-  relevance: PropTypes.number.isRequired
+  relevance: PropTypes.number
```

Comment:
We could leave the `relevance`here (it's needed for the search results), if we adjust the usage in `media-library-search-screen.js` from:

~~~
MediaLibrarySearchScreen.propTypes = {
  files: PropTypes.arrayOf(mediaLibraryItemWithRelevanceShape).isRequired,
  highlightedFile: mediaLibraryItemWithRelevanceShape,
~~~

to:

~~~
MediaLibrarySearchScreen.propTypes = {
  files: PropTypes.arrayOf(mediaLibraryItemWithRelevanceShape).isRequired,
  highlightedFile: mediaLibraryItemShape,
~~~

Reply:
That's better, true, changed it so!